.github/workflows/make-github-and-docker-release.yml

```diff
@@ -0,0 +1,53 @@
+name: Create docker release and github release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Release version number'
+        required: true
+        default: 0.0.0
```

Comment:
Is there a meaning to `default` when `required` is set to `true`?

Reply:
It's a number that appears in the box where we can enter the release number